angular/src/app/component/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css']
+})
+export class DashboardComponent implements OnInit {
+
+
+  constructor(private router: Router) { }
+
+  public user_name: any;
+
+  ngOnInit(): void {
+    this.user_name = localStorage.getItem('username_log');
+  }
+
+  onLogout() {
+    localStorage.setItem('accesstoken', 'hello');
```

Comment:
is this what is the supposed functionality of logout

Reply:
Line 22 is'nt required sir, i forgot to comment it. Line 23 is enough for logout.